### PR TITLE
DPE-5627 Arch support and CC006 compliance

### PR DESCRIPTION
**Pull request by paulomach**
Multi arch support TF "bundle"
Sources charm terraform from charm repo.

Future TODO:
* Parametrized certificates provider (now bundling self-signed)
* Renovate rules for update charm revisions

### CODE REVIEW

File: terraform/applications.tf [hunk 1]
```diff
@@ -0,0 +1,85 @@
+module "postgresql" {
+  source          = "git::https://github.com/canonical/postgresql-operator//terraform?ref=feature/tf-module"
```
Comment (lucasgameiroborges) [inline terraform/applications.tf:2]: `ref=feature/tf-module` will probably become either `main` or revision tag, I imagine?

File: terraform/revisions.tf [hunk 2]
```diff
@@ -0,0 +1,22 @@
+locals {
+  postgresql_revisions = {
+    amd64 = 468,
```
Comment (lucasgameiroborges) [inline terraform/revisions.tf:3]: @taurus-forever : We'll probably need update bundle workflow here, correct? If so, I'll create a ticket for it.

Comment (taurus-forever) [reply terraform/revisions.tf:3]: Yes, now we have too. Please create a followup ticket.

File: terraform/variables.tf [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+variable "postgresql_charm_config" {
+  description = "Postgresql charm configuration"
+  type        = map(string)
+  default     = {}
+}
+
+variable "pgbouncer_charm_channel" {
+  description = "Postgresql router charm channel"
```
Comment (taurus-forever) [inline terraform/variables.tf:75]:
```suggestion
  description = "PgBouncer charm channel"
```

**Review by taurus-forever — APPROVED** [review]
LGTM, but please consider to replace `Postgresql router` :-) and operator git branch.

File: terraform/applications.tf (hunk 1 above)
Comment (paulomach) [reply terraform/applications.tf]: main, done

File: terraform/revisions.tf (hunk 2 above)
Comment (lucasgameiroborges) [reply terraform/revisions.tf:3]: https://warthogs.atlassian.net/browse/DPE-5673

**Pull request merged**